clients/fides-js/src/lib/initialize.ts

```diff
@@ -143,6 +146,35 @@ const automaticallyApplyGPCPreferences = ({
   }
 };
 
+/**
+ * Gets and validates Fides override options provided through URL query params, cookie or window obj.
+ */
+export const getOverrideFidesOptions = (): FidesOptionOverrides => {
```

Comment:
```suggestion
export const getOverrideFidesOptions = (): Partial<FidesOptionOverrides> => {
```

I think this should be a `Partial` since not all the params will be filled out (there's an `if` in the `forEach`. and if it's `typeof window === "undefined` we'll also only get a partial).

Reply:
ah you're right, thanks! that's what I was missing